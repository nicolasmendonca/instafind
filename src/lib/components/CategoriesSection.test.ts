import CategoriesSection from './CategoriesSection.svelte'
import {render} from '@testing-library/svelte'

test('matches the snapshot', () => {
  const {container} = render(CategoriesSection)
  expect(container).toMatchSnapshot()
})
