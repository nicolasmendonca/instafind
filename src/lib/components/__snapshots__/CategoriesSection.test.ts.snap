// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<body>
  <div>
    <section
      aria-label="Categories"
      class="bg-gray-100 grid grid-cols-3 gap-8 px-8 py-8"
    >
      <div
        class="flex flex-col items-center space-y-2"
      >
        <div
          class="h-16 w-16 bg-purple-600 rounded-full"
        />
         
        <h3
          class="text-sm"
        >
          Indumentaria
        </h3>
      </div>
       
      <div
        class="flex flex-col items-center space-y-2"
      >
        <div
          class="h-16 w-16 bg-purple-600 rounded-full"
        />
         
        <h3
          class="text-sm"
        >
          Indumentaria
        </h3>
      </div>
       
      <div
        class="flex flex-col items-center space-y-2"
      >
        <div
          class="h-16 w-16 bg-purple-600 rounded-full"
        />
         
        <h3
          class="text-sm"
        >
          Indumentaria
        </h3>
      </div>
    </section>
  </div>
</body>
`;
